integration-tests/tests/client/process_blocks.rs

```diff
@@ -3601,7 +3605,7 @@ mod storage_usage_fix_tests {
                 ProtocolFeature::FixStorageUsage.protocol_version(),
             );
 
-            let (_, res) = env.clients[0].process_block(block.clone(), Provenance::NONE);
+            let (_, res) = env.clients[0].process_block(block.clone().into(), Provenance::NONE);
             assert!(res.is_ok());
             run_catchup(&mut env.clients[0], &vec![]).unwrap();
 
```

Comment:
Completely unrelated to the PR, but this is a very clear indication that we want some better way to specify testing scenarios, which doesn't need to change every time we do optimization.